This causes illegal instruction faults in pixman, so xserver crashes.
https://github.com/kraj/meta-clang/issues/696

From 8659c5c5bec39dd43a1988b19d4cf30507a44679 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Mon, 28 Nov 2022 16:52:50 +0000
Subject: [PATCH] Revert "pixman: Do not use clang assembler for now"

This reverts commit 84dbafa42d8141b00da75d6664aef07c252a52ee.
---
 conf/nonclangable.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/conf/nonclangable.conf b/conf/nonclangable.conf
index 04112f4..b5db848 100644
--- a/conf/nonclangable.conf
+++ b/conf/nonclangable.conf
@@ -347,5 +347,4 @@ DEPENDS:append:pn-pixman:mips:toolchain-clang = " openmp"
 #| .endfunc
 #| ^
 CFLAGS:append:pn-pixman:arm:toolchain-clang = " -no-integrated-as"
-CFLAGS:append:pn-pixman:aarch64:toolchain-clang = " -no-integrated-as"
 
-- 
2.34.1

